src/daemon/rpc_command_executor.cpp

```diff
@@ -2445,14 +2396,72 @@ bool t_rpc_command_executor::print_sn(const std::vector<std::string> &args)
       return true;
     }
 
+    using hard_fork_height                                         = uint64_t;
+    hard_fork_height volatile hf_heights[cryptonote::network_version_count] = {};
```

Comment:
Any reason to avoid a `std::array<hf_heights, cryptonote::network_version_count>` here instead (primarily to avoid the need for the LOKI_ARRAY_COUNT by being able to use `hf_heights.size()` instead)?

Reply:
No particular reason, my habit. std::array is superior here, has been updated to use that and removed the volatile, that was debugging.